app/src/main/kotlin/me/myshows/android/ui/activities/CommentsActivity.kt

```diff
@@ -37,13 +38,14 @@ class CommentsActivity : HomeActivity() {
 
     private val TAG = CommentsActivity::class.java.simpleName
 
-    private lateinit var client: MyShowsClient
+    @Inject
+    lateinit var client: MyShowsClient
```

Comment:
In Java code this variable is package local, so why in this case the variable isn't `internal`?

Reply:
Because `internal` isn't package local. Read [documentation](https://kotlinlang.org/docs/reference/visibility-modifiers.html).